Sources/Fluent/Model/Collection+Model.swift

```diff
@@ -0,0 +1,35 @@
+import Async
+
+extension Collection where Element: Model, Element.Database: TransactionSupporting {
+    /// Saves a collection of models to Database
+    ///
+    /// - Parameter conn: A means to connect to a database
+    /// - Returns: A future array of the saved models
+    func save(on conn: Element.Database.Connection) -> Future<[Element]> {
+        return self.map { $0.save(on: dbConn) }.flatten(on: dbConn)
```

Comment:
After removing the transaction part, you missed the renaming: `dbConn` should bei `conn`.

```suggestion
        return self.map { $0.save(on: conn) }.flatten(on: conn)
```